src/CodeExamples.sln

```diff
@@ -23,6 +23,40 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMap", "HashMap\HashMap.
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestHashMap", "TestHashMap\TestHashMap.csproj", "{EBDC072A-4291-4482-9BAE-2490C190B1B6}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateUintCapacity", "HashMap.Test\HashMapNamespace.CalculateUintCapacity\HashMapNamespace.CalculateUintCapacity.csproj", "{4DC0AECB-322A-427A-B49B-FA54AF5A8D16}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateCapacity", "HashMap.Test\HashMapNamespace.CalculateCapacity\HashMapNamespace.CalculateCapacity.csproj", "{52CF5877-0FA0-482F-A9E8-137C1FF052EC}"
+# EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ProvideCapacity", "HashMap.Test\HashMapNamespace.ProvideCapacity\HashMapNamespace.ProvideCapacity.csproj", "{6B3046FE-22DE-43B3-9335-83883D159BA1}"
+# EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Add", "HashMap.Test\HashMapNamespace.Add\HashMapNamespace.Add.csproj", "{8513B597-4E9C-44CD-809F-290F446E21E4}"
```

Comment:
### 5/5 Tests Pass.
```
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.Add (roost-1709036480) $ dotnet test
  Determining projects to restore...
  Restored /workspaces/CodeExamples/src/HashMap/HashMap.csproj (in 7.18 sec).
  Restored /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/HashMapNamespace.Add.csproj (in 17.35 sec).
/workspaces/CodeExamples/src/HashMap/HashMap64.cs(214,21): warning CS8601: Possible null reference assignment. [/workspaces/CodeExamples/src/HashMap/HashMap.csproj]
/workspaces/CodeExamples/src/HashMap/HashMap64.cs(220,41): warning CS8604: Possible null reference argument for parameter 'value' in 'HashMap64Node.HashMap64Node(TValue value, long key, int next)'. [/workspaces/CodeExamples/src/HashMap/HashMap.csproj]
/workspaces/CodeExamples/src/HashMap/HashMap64.cs(235,41): warning CS8604: Possible null reference argument for parameter 'value' in 'HashMap64Node.HashMap64Node(TValue value, long key, int next)'. [/workspaces/CodeExamples/src/HashMap/HashMap.csproj]
/workspaces/CodeExamples/src/HashMap/HashMap64.cs(248,40): warning CS8604: Possible null reference argument for parameter 'value' in 'HashMap64Node.HashMap64Node(TValue value, long key, int next)'. [/workspaces/CodeExamples/src/HashMap/HashMap.csproj]
/workspaces/CodeExamples/src/HashMap/HashMap64.cs(295,12): warning CS8601: Possible null reference assignment. [/workspaces/CodeExamples/src/HashMap/HashMap.csproj]
/workspaces/CodeExamples/src/HashMap/HashMap64.cs(326,24): warning CS8601: Possible null reference assignment. [/workspaces/CodeExamples/src/HashMap/HashMap.csproj]
  HashMap -> /workspaces/CodeExamples/src/HashMap/bin/Debug/net6.0/HashMap.dll
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/Add.cs(139,28): warning CS8625: Cannot convert null literal to non-nullable reference type. [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/HashMapNamespace.Add.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/Add.cs(129,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/HashMapNamespace.Add.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/Add.cs(86,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/HashMapNamespace.Add.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/Add.cs(100,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/HashMapNamespace.Add.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/Add.cs(101,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/HashMapNamespace.Add.csproj]
/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/Add.cs(142,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/HashMapNamespace.Add.csproj]
  HashMapNamespace.Add -> /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/bin/Debug/net7.0/HashMapNamespace.Add.dll
Test run for /workspaces/CodeExamples/src/HashMap.Test/HashMapNamespace.Add/bin/Debug/net7.0/HashMapNamespace.Add.dll (.NETCoreApp,Version=v7.0)
Microsoft (R) Test Execution Command Line Tool Version 17.7.2 (x64)
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.

Passed!  - Failed:     0, Passed:     5, Skipped:     0, Total:     5, Duration: 54 ms - HashMapNamespace.Add.dll (net7.0)
vscode ➜ .../CodeExamples/src/HashMap.Test/HashMapNamespace.Add (roost-1709036480) $ 
```